crates/bevy_gizmos/src/config.rs

```diff
@@ -13,6 +13,17 @@ use std::{
     ops::{Deref, DerefMut},
 };
 
+/// An enum configuring how line joins will be drawn.
+#[derive(Debug, Copy, Clone, Reflect, PartialEq, Eq, Hash)]
+pub enum GizmoLineJoint {
+    /// Extends both lines at the joining point until they meet in a sharp point.
+    Miter,
+    /// Draws a round corner with the specified resolution between the two lines.
+    Round(u32),
+    /// Draws a bevel to connect the ends of both lines.
+    Bevel,
+}
```

Comment:
what about we put a None here, so we can avoid using Option when we don't want the joints?

Reply:
It is a bit prettier internally like this because we can just stop the joints rendering process when `None` is passed and can then use `GizmoLineJoint` internally. Although you are right that this might be not that elegant to use.

Reply:
Fair enough. Option is easier to use after all. Probably no problem here.